src/main/java/com/hubspot/jinjava/lib/tag/eager/EagerBlockSetTagStrategy.java

```diff
@@ -31,24 +31,25 @@ protected EagerExecutionResult getEagerExecutionResult(
     String expression,
     JinjavaInterpreter interpreter
   ) {
-    int numEagerTokens = interpreter.getContext().getEagerTokens().size();
-    return EagerReconstructionUtils.executeInChildContext(
-      eagerInterpreter -> {
-        StringBuilder sb = new StringBuilder();
-        for (Node child : tagNode.getChildren()) {
-          sb.append(child.render(eagerInterpreter).getValue());
-        }
-        return EagerExpressionResult.fromString(
-          sb.toString(),
-          numEagerTokens ==
-            eagerInterpreter.getContext().getParent().getEagerTokens().size()
-            ? ResolutionState.FULL
-            : ResolutionState.PARTIAL
```

Comment:
The change is this file is just refactoring to not have to duplicate this ResolutionState selection logic outside of EagerExpressionResolver.java